tools/cross_compile.go

```diff
@@ -0,0 +1,67 @@
+// +build none
```

Comment:
heh, wondered if you'll get caught ;) the +build directive requires an empty line immediately below, before a "regular godoc comment" can start, or it won't work correctly.